### PR TITLE
REGR/DOC: Docs left navbar broke

### LINKED ISSUE #48605: DOC: dev docs left side has no line breaks
https://pandas.pydata.org/pandas-docs/dev/reference/index.html

I'm seeing:

image

I presume this will impact the release, tagging as 1.5. Haven't had a chance to investigate what happened.

### PR DESCRIPTION
**Pull request by rhshadrach**
- [x] closes #48605 (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

cc @phofl

### CODE REVIEW

**Comment by rhshadrach** [comment]
Pages opens with side navigation collapsed:

image

And expands correctly

image

Might be nice to rework the Release notes to take advantage of this - grouping each minor version with drop downs - but certainly outside the scope here

image

File: doc/_templates/sidebar-nav-bs.html [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
-  <div class="bd-toc-item active">
```
Comment (phofl) [inline doc/_templates/sidebar-nav-bs.html:2]: Could you replace the class with ``bd-toc-item navbar-nav``

This works locally and keeps the option to configure differently for different pages

Comment (rhshadrach) [reply doc/_templates/sidebar-nav-bs.html:2]: @phofl - change made, can you double check it's correct? I'm now seeing no dropdowns on the API reference main page:

image

until I click into one

image

Collapsing it, I only get one dropdown

image

I haven't noticed any other changes from removing the file, and this behavior seems worse to me, but it is usable and happy to consider changing after the 1.5 release.

Comment (phofl) [reply doc/_templates/sidebar-nav-bs.html:2]: I am not sure what the impact of collapse=True/False is, so I am careful here.

The dropdown behaviour is consistent with our current docs (1.4.4 and also 1.3) so this should be ok for now I think?

Comment (rhshadrach) [reply doc/_templates/sidebar-nav-bs.html:2]: I didn't realize that it is compatible with those previous docs. Sounds good.

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
Thx @rhshadrach

**Comment by rhshadrach** [comment]
Thanks @phofl for the corrections here.

**Pull request merged**